test_link_checker.py

```diff
@@ -243,29 +254,38 @@ def test_write_response(reset_global):
         assert output_file.readline() == "\n"
         assert output_file.readline() == "by-cc-nd_2.0\n"
         assert output_file.readline() == "URL: https://baseurl/goes/here\n"
-        assert (
-            output_file.readline()
-            == '  Invalid Schema          <a href="file://link3">Invalid Scheme</a>\n'
+        assert output_file.readline() == (
+            "  Invalid Schema          "
+            '<a href="file://link3">Invalid Scheme</a>\n'
         )
-        assert (
-            output_file.readline()
-            == '  400                     <a href="http://httpbin.org/status/400">Response 400</a>\n'
+        assert output_file.readline() == (
+            "  400                     "
+            '<a href="http://httpbin.org/status/400">Response 400</a>\n'
```

Comment:
Any reason for double quotes `"` in the line above (L262) and single quotes `'` here (L263)?